File: Data Structures & Algorithms/longest-consecutive-sequence/submission-1.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        seq = set(nums)
        res = 0

        for num in seq:
            if num - 1 in seq: continue

            curr = 1
            while num + curr in seq: curr += 1

            res = max(res, curr)
        
        return res

File: Data Structures & Algorithms/anagram-groups/submission-1.cpp
class Solution {
public:
    vector<vector<string>> groupAnagrams(vector<string>& strs) {
        vector<vector<string>> res;
        map<vector<int>, vector<string>> mp;
        
        for (string str : strs)
        {
            vector<int> curr(26, 0);

            for (char c : str) curr[c - 'a']++;

            mp[curr].push_back(str);
        }
        
        for (auto& [key, value] : mp) res.push_back(value);

        return res;
    }
};